src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      /* name: 'hay' */
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <Display result="15" />
```

Comment:
- [ ] Since at the Display component we are defining a default value for the result props, there is no need to pass a value for it right now.